aws/mocks/mock_pricing.go

```diff
@@ -0,0 +1,61 @@
+package mocks
+
+import (
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/service/pricing"
+	"github.com/coinbase/odin/aws"
+)
+
+var mockGetProductResponse = `
```

Comment:
nit: can you format this json? It is hard to read

Reply:
Ha, that is werid, looks good in vscode.